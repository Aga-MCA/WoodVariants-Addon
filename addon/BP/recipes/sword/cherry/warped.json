{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "cherry_warped_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "aga_wood_variants:cherry_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:cherry_warped_sword"
    }
  }
}