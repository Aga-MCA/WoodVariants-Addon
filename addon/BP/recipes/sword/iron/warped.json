{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "iron_warped_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "minecraft:iron_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:iron_warped_sword"
    }
  }
}