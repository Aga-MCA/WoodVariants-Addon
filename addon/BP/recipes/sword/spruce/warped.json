{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "spruce_warped_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "aga_wood_variants:spruce_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:spruce_warped_sword"
    }
  }
}