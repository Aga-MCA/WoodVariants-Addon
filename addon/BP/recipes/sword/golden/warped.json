{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_warped_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_warped_sword"
    }
  }
}